### ISSUE TITLE
[Bug] Bio Links do not show when connection to icons.duckduckgo.com fails

### ISSUE THREAD

**Issue by GapVR** [vrcx-team/VRCX#1029]
**Describe the bug**
When connection to icons.duckduckgo.com fails, links in Bio does not show.

**Expected behavior**
None.

**Feature request**
Alt text with full URL is displayed on fail.
